functional-tests/src/test/groovy/data/data_deletion.sql

```diff
@@ -4,10 +4,21 @@ DECLARE
     IDIR_USER varchar = '%bdd-test%';
 BEGIN
     DELETE FROM mine_party_appt WHERE create_user LIKE IDIR_USER;
+    DELETE FROM permit_amendment_document
+    WHERE permit_amendment_id = ANY (
+        SELECT permit_amendment_id FROM permit_amendment
+        WHERE permit_id = ANY (
+        SELECT permit_id FROM permit WHERE create_user LIKE IDIR_USER)
+    );
```

Comment:
Since the functional tests run in their own pod(s) and fail locally when the db isn't empty, I'm curious if we actually need all of these `WHERE` clauses. It looks like the original idea was to allow these tests to create their own records that only they care about and then clean them up after. But since the db essentially needs to be clean before running the tests, it seems like it would be reasonable for the post-run cleanup to do a full cleanup as well.

This would also be simpler to write. Thoughts?

Reply:
It might be simpler.  I think I'll make a ticket for getting the tests working on a non empty database.  https://bcmines.atlassian.net/browse/MDS-2039